src/constants/abacus.ts

```diff
@@ -87,6 +87,8 @@ export type MarketHistoricalFunding = Abacus.exchange.dydx.abacus.output.MarketH
 export type Configs = Abacus.exchange.dydx.abacus.output.Configs;
 export type FeeDiscount = Abacus.exchange.dydx.abacus.output.FeeDiscount;
 export type FeeTier = Abacus.exchange.dydx.abacus.output.FeeTier;
+export type EquityTiers = Abacus.exchange.dydx.abacus.output.EquityTiers;
```

Comment:
[q] just for my education - why are adding equity tiers [to configs](https://github.com/dydxprotocol/v4-abacus/pull/388/files) if we have them as a separate object already?

Reply:
this is based on the abacus bump (local), it didn't exist before